notifications/models.py

```diff
@@ -37,7 +37,7 @@ class Notification(models.Model):
                    'about the notification (e.g. how an achievement is '
                    'unlocked or how long a project report is overdue).'))
     image_url = models.CharField(
-        blank=True,
+        blank=False,
```

Comment:
From Juno via Slack: for the blank=False, I was getting an error when it the field wasn't required, because it might have been set to None. blank=True, null=True might work too